Sources/Services/AWSACM/ACMClient.swift

```diff
@@ -51,6 +51,35 @@ extension ACMClient {
             self.endpointResolver = try endpointResolver ?? DefaultEndpointResolver()
         }
     }
+
+    static func resolve(plugins: [any Plugin]) async throws -> ACMClient.ACMClientConfiguration {
+        let clientConfiguration = try await ACMClient.ACMClientConfiguration()
+        for plugin in plugins {
+            try await plugin.configureClient(clientConfiguration: clientConfiguration)
+        }
+        return clientConfiguration
+    }
+
+    public class Builder {
```

Comment:
Wondering if this Builder can be generic, rather than code-generate it 375 times over.  Looks like every client just needs to have a `Client(config:)` initializer

Reply:
Created `ClientBuilder` class in `smithy-swift` ClientRuntime